Teensy/src/gpio/gpio.cpp

```diff
@@ -94,37 +95,22 @@ static GyroData_s gpio_getGyro()
     return gyro;
 }
 
+// defining resistors for voltage divider
+constexpr float BATTERY_SCALER = (4000.0f + 1000.0f ) / 1000.0f;
+// the ADC analog Input is returned in 10 bit format 
+#define ADC_MAX_VALUE 1023
+// the ADC can only handle 3.3V
+#define ADC_MAX_VOLTAGE 3.3
 static daq_BatteryV_t gpio_getVBat()
 {
-    #include <Arduino.h>
-    // error handling
-    #define NO_STEPPED_DOWN_VOLTAGE 100
-
     // A voltage divider should be scaling down the voltage from 12V to 2.4 V before this
     // refer to https://www.instructables.com/Voltage-Measurement-Using-Arduino/ for how this formula works
```

Comment:
We can probably remove these comments here as these are generally describing the hardware rather than the software :)